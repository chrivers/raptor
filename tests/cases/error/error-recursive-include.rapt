# line 1
INCLUDE test/error-recursive-include.rapt
# line 3