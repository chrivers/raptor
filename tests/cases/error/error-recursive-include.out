error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Error while evaluating INCLUDE
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (included here)
2 |
  |
error: Script Error
 --> tests/cases/error/error-recursive-include.rapt:1:1
  |
1 | INCLUDE error-recursive-include.rapt
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Script error: Too many nested includes
2 |
  |
